pkg/extensions/search/convert/convert_test.go

```diff
@@ -296,6 +298,20 @@ func TestConvertErrors(t *testing.T) {
 	})
 }
 
+func TestUpdateLastUpdatedTimestam(t *testing.T) {
+	Convey("Check image summary when the image has no history", t, func() {
```

Comment:
Call all 3 cases in these tests please (the other tests covering this may change, and we want to be sure we cover).

```
func UpdateLastUpdatedTimestam(repoLastUpdatedTimestamp *time.Time, lastUpdatedImageSummary *gql_generated.ImageSummary,
	imageSummary *gql_generated.ImageSummary,
) *gql_generated.ImageSummary {
	newLastUpdatedImageSummary := lastUpdatedImageSummary

	if repoLastUpdatedTimestamp.Equal(time.Time{}) {
		// initialize with first time value
		*repoLastUpdatedTimestamp = *imageSummary.LastUpdated
		newLastUpdatedImageSummary = imageSummary
	} else if repoLastUpdatedTimestamp.Before(*imageSummary.LastUpdated) {
		*repoLastUpdatedTimestamp = *imageSummary.LastUpdated
		newLastUpdatedImageSummary = imageSummary
	}

	return newLastUpdatedImageSummary
}
```